### ISSUE TITLE
Provide "dump-env" as regular Symfony command

### ISSUE THREAD

**Issue by mbrodala** [symfony/flex#749]
Symfony Flex provides a `dump-env` command which must be invoked using Composer. At its introduction it was suggested to use this in deployment workflows to generate optimized PHP versions of `.env`.

However, this assumes that Composer exists and is executed on remote servers. At least for us this is impossible since our deployment workflows dictate that releases are prepared locally before being uploaded to the remote server. Thus we don't need Composer on the remote server.

Since I assume that we are not alone I would suggest to provide `dump-env` also as regular Symfony command which can be invoked using `bin/console`.

**Comment by stof** [comment]
This should then be implemented in FrameworkBundle rather than in Flex

**Comment by mbrodala** [comment]
Yes, indeed. Are you able to move this ticket to `symfony/symfony` accordingly?
